### ISSUE TITLE
Problem with zooming in 

### ISSUE THREAD

**Issue by TeonTech** [OpenShot/openshot-qt#2978]
Alright let's say you are making a video and you want to zoom in on the top left of the screen. You zoom in on the video editor and it's perfect. You edit the video and then export it into your files. Then you look at your video to make sure it's ok and then look at the part where you zoomed in on the top left of the screen. Even though on the video editor the top left of your screen was in the center and was zoomed in perfectly, now the top left of your screen is at the bottom right of your screen and now 85 percent of your zoomed in part is just the outside black screen of the video. Basically, the video editor over zooms your video to far up, down, left or right and you have to zoom in not perfectly and predict how it will look when the video is exported. Please fix this bug it's really annoying and makes my videos look worse.

**Comment by ferdnyc** [comment]
Hm. I can't immediately reproduce. It's worth noting that OpenShot doesn't _have_ a zoom feature, per se — what it _does_ allow you to do is enlarge the video in the output frame. That's a minor-seeming but actually important distinction, in terms of what's actually changing size / moving.

Anyway, here's what I did:

1. Set up a new project, profile `HD 720p 30 fps`
2. Imported a video. In my case, it was actually larger than the project dimensions, `1920x1080`, in case that turns out to matter.
3. Added the clip to the timeline.
4. Selected the clip, made sure Scale was set to `Best Fit`, and raised `Scale X` and `Scale Y` both to `2.0`.
5. Activated Transform, and repositioned the enlarged video so that the upper left corner was in the frame. (In this case, that corresponded to `Location X = 0.48`, `Location Y = 0.23`.)
6. Exported the video to MP4, medium quality
7. Played the video in VLC

What I saw in VLC exactly matched what I saw in the OpenShot preview, in terms of scaling and centering.

If you're seeing different results, we'll probably need a sample project file and (input) video demonstrating the problem.
